apps/curation_front_end/src/pages/curations/index.tsx

```diff
@@ -3,56 +3,62 @@
 // SPDX-License-Identifier: MIT
 
 import { useUser } from "@/hooks/useUser";
+import useSettingsStore from "@/store/settings.store";
 import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
 import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
 import BulkCurationList from "@/components/bulk_curation_table/BulkCurationList";
 import LicenseConclusionList from "@/components/license_conclusion_table/LicenseConclusionList";
 import PathExclusionList from "@/components/path_exclusion_table/PathExclusionList";
 
 export default function CurationsLibrary() {
-    const user = useUser({});
+    const user = useUser({ redirectTo: "/login", redirectIfFound: false });
+    useSettingsStore.setState({ user: user });
```

Comment:
At least at first glance there seems to be different ways get user - here the return value of `useUser` is set, but in other places the value from Settings Store is used. It would probably make sense to only get user with one of the methods.